nextflow.config

```
@@ -267,7 +266,7 @@ manifest {
     mainScript      = 'main.nf'
     defaultBranch   = 'main'
     nextflowVersion = '!>=23.10.1'
-    version         = '1.1.8'
+    version         = '1.1.8 + Himut'
```

There can't be spaces
```suggestion
    version         = '1.1.9-himut.1'
```

---

Got it!